clang/test/CIR/CodeGen/attributes.c

```diff
@@ -5,11 +5,11 @@ extern int __attribute__((section(".shared"))) ext;
 int getExt() {
   return ext;
 }
-// CIR:   cir.global "private" external @ext : !s32i {section = ".shared"}
+// CIR:   cir.global "private" external @ext : !s32i {alignment = 4 : i64, section = ".shared"}
```

Comment:
Note: we usually try to make the attributes be first class in the operation, this is fine for `alignment` because it's already the case in many tests.